config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -181,3 +181,69 @@ presets:
   # Increase delete collection parallelism.
   - name: TEST_CLUSTER_DELETE_COLLECTION_WORKERS
     value: --delete-collection-workers=16
+
+- labels:
```

Comment:
It's not the most beautiful approach, because there is huge duplication between this and regular docker one, but I want to have those tests running to see the difference.

Reply:
Could you document that in a TODO, cleaning it up could be a good first issue.